### DISCUSSION TITLE
questions for imodeljs

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1294] [Q&A] [answered]
1.now  the map in imodeljs is bing map,i want load local map data from local server ,format is tif 、tmp 、 jpg
2.change by iTwin Snapshot， in revit ,the Geographical coordinates must set Latitude and longitude ,but some time the revit file set value in Project base point (and the value have other Coordinate system)

about  second question the Project base point pic like is :
![QQ截图20210428173425](https://user-images.githubusercontent.com/18022720/116382483-89539500-a848-11eb-9a71-050ddda95806.jpg)

**Comment by pmconne** [comment, marked as answer]
iModel.js cannot know how to query your local map server or interpret its data. Consider writing a [TiledGraphicsProvider](https://github.com/imodeljs/imodeljs/blob/master/core/frontend/src/tile/TiledGraphicsProvider.ts).

Please post separate topics for the other questions, ideally providing more details than you have provided here.
